code/modules/holiday/holidays.dm

```diff
@@ -208,7 +230,15 @@
 	drone_hat = /obj/item/clothing/mask/facehugger/dead
 
 /datum/holiday/UFO/getStationPrefix() //Is such a thing even possible?
-	return pick("Ayy","Truth","Tsoukalos","Mulder") //Yes it is!
+	return pick("Ayy","Truth","Tsoukalos","Mulder","Scully") //Yes it is!
+
+/datum/holiday/USA
```

Comment:
nanotrasen doesn't care about nations, does this really fit?

Reply:
There are liberation station vendors and centcom was burger flag colored for the 4th of July, I think it fits. Nothing I feel strongly about either way though

Reply:
![image](https://user-images.githubusercontent.com/29008542/43690286-da32a87e-98d5-11e8-86ce-61976a24056a.png)
i mean we already have plenty of refrences to america so i think its fine

Reply:
except the headcoder isn't american now.

The oceaniac reign of terror begins

Reply:
yes